### DISCUSSION TITLE
Doris HTTP API

### DISCUSSION THREAD

**Discussion by 16601746291** [apache/doris#18370] [A - General / Q&A] [answered]
curl -X GET http://127.0.0.1:8040/api/compaction/show?tablet_id=464374

response eg:

{
    "status": "Fail",
    "msg": "convert tablet_id and schema_hash failed, stoul"
}

what means?  not yet Tablet not found ,somebody can answer?

**Comment by caoliang-web** [comment, marked as answer]
What version is doris ？ Confirm the tablet is correct

**Reply by 16601746291** [reply]
It is my Doris version too old

**Comment by yagagagaga** [comment]
Is that your tablets are missing or Unhealthy? You can use `show tablets from table_name` to view more information.
<img width="636" alt="截屏2023-04-17 14 06 44" src="https://user-images.githubusercontent.com/21151126/232398597-ed4b5ac7-27b6-4627-a633-dce7691b3c07.png">

**Reply by 16601746291** [reply]
thanks ，it is my version is too old Doris1.0.0,I upgrade my Doris version to 1.2.3
